src/lib.rs

```diff
@@ -142,3 +142,6 @@ mod sealed {
     // ```
     pub trait Sealed {}
 }
+
+const PERLASM_CONFIGURED: () =
+    assert!((cfg!(perlasm) && !cfg!(no_perlasm)) || (!cfg!(perlasm) && cfg!(no_perlasm)));
```

Comment:
I couldn't find a more appropriate place for this assertion. What do you think about its placement?

Reply:
Oops, I just noticed that you specifically suggested we add this assertion to `lib.rs`. Marking as resolved.